src/backend/CommandBufferStateTracker.cpp

```diff
@@ -61,7 +61,7 @@ namespace backend {
 
     bool CommandBufferStateTracker::ValidateCanDispatch() {
```

Comment:
We should check that we aren't in a compute pass at the end of ValidateGetResult.